presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -157,7 +159,12 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
                 schema.getFields().stream()
                         .map(Conversions::toColumnMetadata)
                         .collect(toImmutableList());
-        return new ConnectorTableMetadata(schemaTableName, columns);
+
+        ImmutableMap.Builder<String, Object> properties = ImmutableMap.builder();
```

Comment:
Given there are other properties (clustering) and other may be added in the future, it is possible to move this code into a `BigQueryTableProperties.getTableProerties(TableDefinition)` method, that will return the a map with all the properties, leaving `BigQueryMetadata.getTableMetadata()` cleaner.